src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,130 @@
+package duke.util;
+
+import duke.task.Task;
+
+/**
+ * Represents user interface of the chatbot.
+ */
+public class Ui {
+    private static final String HORIZONTAL_LINE = "____________________________________________________________\n";
+    private String message;
+
+    /**
+     * Constructor for UI class.
+     */
+    public Ui() {
+        this.message = "";
+    }
+
+    /**
+     * Prints the entry message of the chatbot.
+     */
+    public String printEntryMessage() {
```

Comment:
I think can either 
utilize this method to set the welcome message of your GUI since you're returning a string here! (can consider making this method static)
or if not can remove for cleanliness